Source/2_Core/Replayer/ReplayerMenuLoader.cs

```diff
@@ -140,28 +145,39 @@ public enum LoaderState {
         [Inject, UsedImplicitly]
         private readonly IFPFCSettings _fpfcSettings = null!;
 
-        private void StartReplay(Player player) { 
-            StartReplayAsync(Replay!, player, ConfigFileData.Instance.ReplayerSettings);
+        [Inject, UsedImplicitly]
+        private readonly BeatmapLevelsModel _levelsModel = null!;
+
+        private void StartReplay(Player player) {
+            StartReplayAsync(Replay!, player, null);
         }
 
-        public async void StartReplayAsync(Replay replay, Player? player = null, ReplayerSettings? settings = null) {
-            settings ??= ConfigFileData.Instance.ReplayerSettings;
-            var data = new ReplayLaunchData();
-            data.Init(replay, settings, player);
-            await StartReplayAsync(data);
+        [UsedImplicitly]
+        public async void StartReplayAsync(Replay replay, Player player, ReplayerSettings? settings = null) {
+            await StartReplayAsync(replay, player, settings, CancellationToken.None);
         }
 
-        public async Task StartReplayAsync(ReplayLaunchData data) {
+        [UsedImplicitly]
```

Comment:
use `[UsedImplicitly]` attribute only for implicit usage (by reflection for example)
use `[PublicAPI]` attribute for public methods meant to be used from other mods